config.toml

```diff
@@ -67,13 +67,28 @@ url = "https://eksworkshop.com/"
 weight = 21
 
 [[menu.shortcuts]]
-name = "<i class='fas fa-chalkboard-teacher'></i> ECS Workshop"
-url = "https://ecsworkshop.com/"
-weight = 22
+name = "<i class='fas fa-chalkboard-teacher'></i> Learn more about Kubecost and AWS collaboration"
+url = "https://www.kubecost.com/aws-eks-cost-monitoring/"
+weight = 23
+
+[[menu.shortcuts]]
+name = "<i class='fas fa-chalkboard-teacher'></i> Kubecost documentation for Amazon EKS"
+url = "https://docs.kubecost.com/install-and-configure/install/provider-installations/aws-eks-cost-monitoring"
+weight = 23
+
+[[menu.shortcuts]]
+name = "<i class='fas fa-chalkboard-teacher'></i> AWS Cloud billing integration"
+url = "https://docs.kubecost.com/install-and-configure/install/cloud-integration/aws-cloud-integrations"
+weight = 23
+
+[[menu.shortcuts]]
+name = "<i class='fas fa-chalkboard-teacher'></i> Video - EKS Cost Monitoring with Kubecost"
```

Comment:
do we need both Containers from the couch episodes? We could probably just have the latest one covering EKS integrations.

Reply:
I think it is good to have both of them as the first one also cover Kubecost API, Kubecost CLI tool in depth which could also benefit AWS customers

Reply:
@youngjeong46 I updated the video description and remove the 2nd link from CFTC. We can add it later if it is necessary.